charts/internal/cri-resmgr-installation/templates/installation-service.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2022 Intel Corporation. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+apiVersion: v1
+kind: Service
```

Comment:
Do we still need to use "service" even when "prometheus service discovery" role is node? I thought those are two oppossing solutions

Reply:
I understand now - we use role service/endpoint because we don't connect node port directly but by apiserver proxy.